var/spack/repos/builtin/packages/mpich/package.py

```diff
@@ -41,11 +42,25 @@ class Mpich(AutotoolsPackage):
     version('3.1.1', '40dc408b1e03cc36d80209baaa2d32b7')
     version('3.1',   '5643dd176499bfb7d25079aaff25f2ec')
     version('3.0.4', '9c5d5d4fe1e17dd12153f40bc5b6dbc0')
+    version('develop', git='git://github.com/pmodels/mpich')
 
     variant('hydra', default=True,  description='Build the hydra process manager')
     variant('pmi',   default=True,  description='Build with PMI support')
     variant('romio', default=True,  description='Enable ROMIO MPI I/O implementation')
-    variant('verbs', default=False, description='Build support for OpenFabrics verbs.')
+    variant(
+        'device',
+        default='ch3',
```

Comment:
These weren't set before so I'm curious what the defaults are when they aren't set.

Reply:
ch3 is the default "device" in all the MPICH versions in this file.

Reply:
And "tcp" is the default network module when using the ch3:nemesis channel. If you are using the ch3:sock channel, then there are no network modules, only tcp/unix sockets. I resisted putting a "channel" variant in here because the ch4 device implementation does not have a channel abstraction. Only netmods.